spec.md

```diff
@@ -676,9 +676,8 @@ message ValidateVolumeCapabilitiesRequest {
   // The API version assumed by the CO. This is a REQUIRED field.
   Version version = 1;
 
-  // The information about the volume to check. This is a REQUIRED
-  // field.
-  VolumeInfo volume_info = 2;
+  // The ID of the volume to check. This is a REQUIRED field.
+  VolumeID volume_id = 2;
```

Comment:
It seems to me that the other fields in `VolumeInfo` except for `id` and `metadata` are not necessary to determine `VolumeCapability`. As of the stateless property, most of the other API functions ask for two parameters: `VolumeID` and `VolumeMetadata`, instead of a single `VolumeInfo`. For consistency, I suggest that we use two parameters: `VolumeID` and `VolumeMetadata` here, or we could change the other functions to accept a `VolumeInfo` parameter. Thoughts?